docs/user-guide/garbage-collector.md

```diff
@@ -0,0 +1,21 @@
+---
+---
+
+* TOC
+{:toc}
+
+## WARNING: Garbage Collector is an alpha feature, use it at your own risk!
+
+### What does the garbage collector do
+
+The garbage collector (GC) monitors the cluster and checks the metadata.ownerReferences field of each object. If all the owners present in metadata.ownerReferences do not exist in the cluster, the GC will request the API server to delete the object.
```

Comment:
If any the owners present in metadata.ownerReferences do not exist in the cluster, the GC will delete the object.